src/libraries/System.Net.Security/tests/StressTests/SslStress/run-docker-compose.ps1

```diff
@@ -15,6 +15,13 @@ Param(
 $REPO_ROOT_DIR = $(git -C "$PSScriptRoot" rev-parse --show-toplevel)
 $COMPOSE_FILE = "$PSScriptRoot/docker-compose.yml"
 
+# This is a workaround for an issue with 1es-windows-2022-open, which should be eventually removed.
+# See comments in <repo>/eng/pipelines/libraries/stress/ssl.yml for more info.
+$dockerComposeCmd = $env:DOCKER_COMPOSE_CMD
+if (!(Test-Path $dockerComposeCmd)) {
+    $dockerComposeCmd = "docker-compose"
+}
```

Comment:
This is now an asymmetry between `run-docker-compose.ps1` in SslStress vs HttpStress.
For some reason, the HttpStress pipeline is ok to `docker-compose build` with V1 of compose but then run with V2, however in SslStress this resulted in a failure.
I don't want to add an equivalent workaround to the `.ps1` in HttpStress, when it's not necessary.